### PR TITLE
[1.5.z] Release version 1.5.3

### PR DESCRIPTION
**Pull request by mocenas**
### Summary

Release version 1.5.3

Please check the relevant options

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dependency update
- [ ] Refactoring
- [X] Release (follows conventions described in the [RELEASE.md](https://github.com/quarkus-qe/quarkus-test-framework/blob/main/RELEASE.md))
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Example scenarios has been updated / added
- [ ] Methods and classes used in PR scenarios are meaningful
- [ ] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
@mocenas how much you are in hurry with this release? I forgot that this https://github.com/quarkus-qe/quarkus-test-framework/pull/1320 needs to be backported to 1.5.z and bumped in TS otherwise one test will be flaky there. but it's not super urgent, I can make another release later this week.

TL;DR; if you are not in hurry, let's backport https://github.com/quarkus-qe/quarkus-test-framework/pull/1320 first and rebase this PR on next  1.5.z

**Comment by michalvavrik** [comment]
opened https://github.com/quarkus-qe/quarkus-test-framework/pull/1327 but still, if this is urgent, I can wait

**Comment by mocenas** [comment]
> @mocenas how much you are in hurry with this release? I forgot that this #1320 needs to be backported to 1.5.z and bumped in TS otherwise one test will be flaky there. but it's not super urgent, I can make another release later this week.
> 
> TL;DR; if you are not in hurry, let's backport #1320 first and rebase this PR on next 1.5.z

I would like to release it today. If you can merge #1327 soon, I have no problem rebasing this on it.

**Comment by michalvavrik** [comment]
> > @mocenas how much you are in hurry with this release? I forgot that this #1320 needs to be backported to 1.5.z and bumped in TS otherwise one test will be flaky there. but it's not super urgent, I can make another release later this week.
> > TL;DR; if you are not in hurry, let's backport #1320 first and rebase this PR on next 1.5.z
> 
> I would like to release it today. If you can merge #1327 soon, I have no problem rebasing this on it.

cool, I think backport PR should be fine, it depends on OCP CI which I can't influence.

**Comment by michalvavrik** [comment]
@mocenas let's rebase, let format validation pass and merge this (no point waiting, we just run GH/OCP CI in #1327)

**Review by michalvavrik — APPROVED** [review]
Please don't forget to rebase.

**Comment by mocenas** [comment]
> Please don't forget to rebase.

Rebased.

IDK do we need to run CI again?

**Comment by michalvavrik** [comment]
> IDK do we need to run CI again?

NO!
